### DISCUSSION TITLE
Use sftpgo as a module

### DISCUSSION THREAD

**Discussion by soumeny2k** [drakkan/sftpgo#1687] [Q&A] [answered]
Our organization uses lots of custom in house product. To avoid future maintenance issue and sftpgo feature requests, I want to use sftpgo as a module in my golang project and build customization on top of sftpgo sftp server. Can I do that? I have tried to pull it as a module, but looks like that is not working. If that isn't an option, then any other alternative solution possible?

**Comment by drakkan** [comment, marked as answer]
SFTPGo is a server application, not a library. Also, by including SFTPGo code in your application you are creating a derivative work. See our [compliance](https://sftpgo.com/compliance.html) page to better understand what this means. Thank you

**Reply by soumeny2k** [reply]
ok that make sense. thank you
